paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheet.kt

```diff
@@ -181,6 +181,8 @@ class CustomerSheet @Inject internal constructor(
         val preferredNetworks: List<CardBrand> = emptyList(),
 
         internal val allowsRemovalOfLastSavedPaymentMethod: Boolean = true,
+
+        internal val paymentMethodOrder: List<String> = emptyList(),
```

Comment:
similar question to the default value question I asked in PaymentMethodOrderSettingDefinition -- should we represent the default value as null instead of an empty list? I think using empty list makes sense if that's a valid value for this parameter (which I think it is), but just double checking

Reply:
I think empty list is ideal here. If this list is missing values that exist in the payment intent, we use the order that comes back from the payment intent. So I think emptyList represents that same behavior ideally.